readthedocs/core/symlink.py

```diff
@@ -243,7 +241,11 @@ def symlink_single_version(self):
         Link from $WEB_ROOT/<project> ->
                   HOME/user_builds/<project>/rtd-builds/latest/
         """
-        default_version = self.project.get_default_version()
+        default_version = self.get_default_version()
+        if default_version is None:
+            self._log("Default version wrong privacy level.")
```

Comment:
Okay, yeah, I'd say debug then, if it's just a normal case we expect to hit -- or nothing actionable is required on its notice.